utils/buildcli/buildcli/clang.py

```diff
@@ -0,0 +1,85 @@
+from __future__ import absolute_import
```

Comment:
Please add Swift license headers to all new files with code (you can copy from `build-script`).